using/solving-exercises/README.md

```diff
@@ -0,0 +1,21 @@
+# Solving Exercises
+
+The primary way in which Exercism helps you learn is by solving exercises.
+
+There are two ways to solve exercises on Exercism:
+
+1. [Online via the website's in-browser editor](./online.md)
+2. [On your local machine via the CLI](./locally)
+
+## Which option should I choose?
+
+There are different trade-offs between using the online editor or CLI:
+
+|            | Online editor                          | CLI                             |
+| ---------- | -------------------------------------- | ------------------------------- |
+| Installing | No                                     | Yes (e.g. SDK)                  |
+| Editing    | Browser                                | Your preferred IDE              |
+| Testing    | Run tests on website                   | Run tests locally               |
+| Debugging  | Limited support (only for some tracks) | Full support (depending on IDE) |
```

Comment:
I'd also consider removing the reference to "IDE" here too, since people might prefer to debug without using an IDE.

Reply:
Maybe:

```suggestion
| Debugging  | Limited support (only for some tracks) | Full support (depending on your editor) |
```

Reply:
Mmm. I think I prefer something like:
> Full support (depending on your machine)

> Full support (depending on your environment)

But I'm fine with whatever you want here.

Reply:
> Full support (depending on your environment)

This is the best option I think. I'll update